### ISSUE TITLE
Dwarves in torches room in BRD respawning a little too fast

### ISSUE THREAD

**Issue by arooks1** [Wall-core/Everlook-Bugtracker#463]
After running BRD, we all noticed that the low hp dwarves in the torches room in BRD, before Magmus, are respawning a little too fast. 

Also, after we opened both doors, the dwarves were still respawning, which I think they shouldn't

**Comment by Wall-core** [comment]
Respawn timers are correct. Did you open the door using two torches on one platform? If so, that's likely why they continued respawning.

**Comment by arooks1** [comment]
No, we used 1 torch on each platform. However, we did wipe between them.
